Support API v2 and Make Raw Request (#19)
elasticio/twitter-component: README.md

# Twitter Component

## Table of Contents

* [Description](#description)
* [Credentials](#credentials)
* [Actions](#actions) 
  * [Post tweet](#post-tweet) 
  * [Make Raw Request](#make-raw-request)
* [Triggers](#triggers)

## Description

Twitter Component is designed to connect with Twitter API v2

## Credentials

> To authenticate component you will need to create App in [developer portal’s App page](https://developer.twitter.com/en/portal/projects-and-apps), component use OAuth 1.0a. [more info](https://developer.twitter.com/en/docs/apps/overview)

Reviewer: ```suggestion
To authenticate the component you will need to create an App in [developer portal’s App page](https://developer.twitter.com/en/portal/projects-and-apps), the component uses OAuth 1.0a. [more info](https://developer.twitter.com/en/docs/apps/overview)

```

Component credentials configuration fields: 
* **Consumer key**  (string, required) - Consumer API key
* **Consumer secret**  (string, required) - Consumer API secret
* **Access token key**  (string, required) - Authentication access token
* **Access token secret**  (string, required) - Authentication access secret

## Actions 

### Post tweet 

Post new tweet

#### Configuration Fields

none

#### Input Metadata

* **Text** - (string, required): Text of your tweet

#### Output Metadata

> * **id** - (string, required): Unique identifier for your tweet

Reviewer: ```suggestion
* **id** - (string, required): Unique identifier of your tweet

```

* **text** - (string, required): Text of your tweet 
* **edit_history_tweet_ids** - (array, required): list of tweet ids edit history
  
### Make Raw Request 

Executes custom request.

#### Configuration Fields

none

#### Input Metadata

> * **Url** - (string, required): Path of the resource. ex: `/2/users/me`

Reviewer: ```suggestion
* **Url** - (string, required): Path of the resource. e.g.: `/2/users/me`

```
